app/views/content/how-to-apply-for-teacher-training/teacher-training-personal-statement.md

```diff
@@ -57,7 +57,16 @@ You could include:
 
 Teacher training providers want to see your passion and that you understand the bigger picture of teaching.
 
-Make sure you check your spelling and grammar in your application. You want to make the best possible impression.
+### How to write your personal statement
+
+When writing your personal statement you should make sure you check your spelling and grammar in your application. You want to make the best possible impression.
+
+You can use ChatGPT or other artificial intelligence (AI) tools to help you write your personal statement. You should not rely on it to write your entire statement because:
+
+* AI tools use bland language and will not be able to give details about you as a person and using them may result in your application being unsuccessful
+
+* your account to apply for teacher training may be blocked if you consistently submit personal statements that look like they have been written with AI tools
```

Comment:
@caitlin-reidy Should Apply be capped?

Reply:
I'm not sure whether it should unless we are saying `your Apply for teacher training account`?

But I think our style guide says to avoid using names of services directly?

Reply:
I think leave as uncapped because of the order of the sentence